app/commands/mysql/subcommand/import.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/sh
+
+# Source functions library.
+. ${BASE}/functions
+
+DST_DIR="/tmp"
+MIME_EXPECTED="text/plain"
+
+# Check for the required input
+if [ -z "${DB_HOST}" ]; then
+  echo "You have to define the db host"
+  exit 12
+fi
+if [ -z "${DB_USER}" ]; then
+  echo "You have to define the db user"
+  exit 12
+fi
+if [ -z "${DB_PASSWORD}" ]; then
+  echo "You have to define the db password"
+  exit 12
+fi
+if [ -z "${DB_NAME}" ]; then
+  echo "You have to define the db name"
+  exit 12
+fi
+if [ -z "${DB_PORT}" ]; then
+  echo "You have to define the db port"
+  exit 12
+fi
+if [ -z "${PROVIDER}" ]; then
+  echo "You have to define the bucket provider"
+  exit 12
+fi
+if [ "${PROVIDER_LOWER}" != "aws" ] && [ "${PROVIDER_LOWER}" != "gcp" ] && [ "${PROVIDER_LOWER}" != "minio" ]; then
+  echo "You have to define a valid bucket provider (aws, gcp, minio)"
+  exit 12
+fi
+if [ "${PROVIDER_LOWER}" = "minio" ] && [ -z "${BUCKET_ENDPOINT}" ]; then
+  echo "You have to define the bucket endpoint"
+  exit 12
+fi
+if [ -z "${BUCKET}" ]; then
+  echo "You have to define the bucket name"
+  exit 12
+fi
+if [ -z "${FILE}" ]; then
+  echo "You have to define the file in the bucket"
+  exit 12
+fi
+
+# All the required inputs are present! Do the job
+debug "All the required inputs are present. Go on with the real job."
+
+debug "Downlaod the file from AWS bucket."
+
+if [ "${PROVIDER_LOWER}" = "aws" ]; then
+  rclone \
+    --s3-provider=AWS \
+    --s3-region="${AWS_DEFAULT_REGION}" \
+    --s3-location-constraint="${AWS_DEFAULT_REGION}" \
+    --s3-access-key-id=${AWS_ACCESS_KEY_ID} \
+    --s3-secret-access-key=${AWS_SECRET_ACCESS_KEY} \
+    copy :s3://${BUCKET}/${FILE} "${DST_DIR}" 2> /dev/null
+elif [ "${PROVIDER_LOWER}" = "gcp" ]; then
+  rclone \
+    copy :gcs://${BUCKET}/${FILE} "${DST_DIR}" 2> /dev/null
+elif [ "${PROVIDER_LOWER}" = "minio" ]; then
+  rclone \
+    --s3-provider=Minio \
+    --s3-endpoint=${BUCKET_ENDPOINT} \
+    --s3-access-key-id=${MINIO_ACCESS_KEY} \
+    --s3-secret-access-key=${MINIO_SECRET_KEY} \
+    copy :s3://${BUCKET}/${FILE} "${DST_DIR}" 2> /dev/null
+fi
```

Comment:
Would be nice to have an ADDITIONAL_ARGS variable here to be able to extend from the outside the rlone command

Reply:
I have added the `--rclone-add-params` switch or `RCLONE_ADD_PARAMS` env var to allow the users to use the additional parameters